power-platform/admin/managed-environment-enable.md

```diff
@@ -51,6 +51,23 @@ Use the following settings to increase visibility and control for the selected e
 | **[Data policies](managed-environment-data-policies.md)** | Help safeguard your organizational data by limiting the connectors available.|
 | See active data policies for this environment | View the policies that define the consumer connectors that specific data can be shared with. |
 
+## Enable Managed Environments using PowerShell
+
+Admins can use PowerShell to enable the Environments property for an environment.
```

Comment:
Admins can also use PowerShell to enable Managed Environments. Below is a PowerShell script that enables it for a single environment: